src/puppetlabs/ring_middleware/common.clj

```diff
@@ -48,6 +49,5 @@
                             http-opts)
                      request
                      prepare-cookies)]
-    (log/debug "Proxying request to" (:uri req) "to remote url" (str remote-uri)
-               ". Remote server responded with status" (:status response))
+    (log/debug (trs "Proxying request to {0} to remote url {1}. Remote server responded with status {2}" (:uri req) (str remote-uri) (:status response)))
```

Comment:
nitpick: don't worry about this for this PR, but, in the future, would be good to put some newlines in for lines that go over 80 char.  I assume that for the trs stuff we have to be careful about not breaking up the strings, but the args could wrap to the next line.

no biggie though.